utHelper.py

```diff
@@ -132,43 +132,6 @@ def changeFolderPermission(self, permission, path, session=None):
         session.write("chmod " + permission + " " + path + "*")  # Send the 'chmod' command
         session.write("\n")  # Send a newline
 
-    def copyFileFromHost(self, sourcePath, destinationPath, targetDevice="dut"):
```

Comment:
you've removed this function, Are we sure it doesn't still exist? You would have to check all use cases for it and it's not just your team that's using it. Raju's team are using it too.

There's two options for it to still exist, either make it a wrapper functions.

```python
copyFileFromHost()
{
   self.utils.copyFileFromHost( blah )
}
```

or you can inherit both classes during the init, so anything in `utBaseUtils` is still present or avaialble for anyone using `utHelperClass`

```yaml
class utHelperClass(testController,utBaseUtils):
```

## Example of multiple Inheritance

```python
class Animal:
  def __init__(self, name):
    self.name = name

class Flyer:
  def __init__(self, wingspan):
    self.wingspan = wingspan

class Bird(Animal, Flyer):
  def __init__(self, name, wingspan, species):
    Animal.__init__(self, name)
    Flyer.__init__(self, wingspan)
    self.species = species

  def fly(self):
    print(f"{self.name} the {self.species} is flying with a {self.wingspan} wingspan!")

# Create a Bird object
my_bird = Bird("Tweety", "10 inches", "Canary")
my_bird.fly()  # Output: Tweety the Canary is flying with a 10 inches wingspan!
```

Reply:
assumed that utHelper was only used by us (VTS tests), so I removed it. I will keep the function that performs the same task but uses the utBaseutils scpCopy function